data/binding/comparator_helper.go

```diff
@@ -1,6 +1,8 @@
 package binding
 
-import "fyne.io/fyne/v2"
+import (
+	"fyne.io/fyne/v2"
+)
```

Comment:
This seems like an unnecessary change. I think it is a leftover from the wrapper that was removed.

Reply:
Oh yes sorry about that.